lib/vagrant-openshift/command/test_openshift3_image.rb

```diff
@@ -103,31 +103,20 @@ def execute
 # switch to the desired ref
 git checkout #{ref}
 
-# If software version is set, change directory to it
-[ -n "#{image_version}" ] && pushd #{image_version} > /dev/null
+if [ #{image} != "sti-base" ]; then
+  # Pull base images
+  docker pull ci.dev.openshift.redhat.com:5000/openshift/base-centos7 && docker tag ci.dev.openshift.redhat.com:5000/openshift/base-centos7 openshift/base-centos7
+  docker pull ci.dev.openshift.redhat.com:5000/openshift/base-rhel7 && docker tag ci.dev.openshift.redhat.com:5000/openshift/base-rhel7 openshift/base-rhel7
+fi
 
-# Build the STI image we use for testing
-docker build -t #{image}-candidate .
+# Test CentOS 7 based images with STI test framework
+make test VERSION=#{image_version}
 status=$?
 
-# Run the STI image test framework
 if [ $status -eq 0 ]; then
-  IMAGE_NAME=#{image}-candidate ./test/run
+  # Test RHEL 7 based images with STI test framework
+  make test TARGET=rhel7 VERSION=#{image_version}
   status=$?
-
-  # If software version is set, go back to the repository root
-  [ -n "#{image_version}" ] && popd > /dev/null
-fi
-
-if [ $status -eq 0 ]; then
-  # get the image id
-  image_id=`docker inspect --format="{{ .Id }}" #{image}-candidate:latest`
-
-  # tag it devenv-ready
-  docker tag $image_id #{image}:devenv-ready
```

Comment:
Is devenv-ready really appropriate as a new term?  We have qe-ready.  Are these images targeted to be ready just for devenvs?